ones-and-zeros.py

```diff
@@ -0,0 +1,7 @@
+def binary_array_to_number(arr):
+  """Given list of 1s and 0s, convert equivalent binary value to an integer."""
+  in_string = []
```

Comment:
These three lines can be replaced with a simple list comprehension. Make sure you know how to use list comprehensions, because you'll see them a lot!
`in_string = [str(num) for num in arr]`